projects/storefrontapp-e2e-cypress/cypress/integration/regression/scrolling/scroll-position-restoration.e2e-spec.ts

```diff
@@ -0,0 +1,36 @@
+context('scroll Position Restoration', () => {
+  const PRODUCT_NAME = 'DC Car Battery Adapter';
```

Comment:
Can default product be used instead? 
Less maintenance costs afterwards.

Reply:
@giancorderoortiz 
We need to select a product in the middle of the screen to check scrolling.
Thanks